2018/_data/menu.json

```diff
@@ -35,17 +31,7 @@
     },
     {
       "label": "Travel",
-      "href": "/#Travel",
-      "items": [
-        {
-          "label": "Accomodation",
-          "href": "/accomodation.html"
-        },
-        {
-          "label": "To Barcelona",
-          "href": "/travel-to-java-barcelona.html"
-        }
-      ]
+      "href": "/travel-to-java-barcelona.html"
```

Comment:
Why you have removed the accomodation page? Then the people won't see the deal we have with Alimara

Reply:
Not in purpose, apparently something happened with pulling from branches